### DISCUSSION TITLE
When are gain ref flips and/or rotations needed

### DISCUSSION THREAD

**Discussion by JuhaHuiskonen** [nextpyp/.github#59] [Q&A] [answered]
My gain reference is in `.gain` format and movies are `.eer`.  Do I need to rotate or flip the axis? In general, in what circumstances are these options needed?

**Comment by yezhou867** [comment]
If you rename the .gain to .tif, nextPYP should recognize the gain reference.  Certain operations are needed when you see the streaks crossing the overall image with repeated patterns in the import block image or preprocessing images.  But sometimes it's hard to tell the right operation directly from the image. If no info from data collection, trial and error could be used until the streaks disappear.

**Reply by yezhou867** [reply]
No renaming is needed in the new version.

**Reply by abartesaghi** [reply, marked as answer]
nextPYP supports the use of gain references in .gain format starting with [v0.6.1](https://nextpyp.app/files/pyp/0.6.1/docs/index.html).
